src/MailingListBundle/Block/Service/ContactsManager.php

```diff
@@ -0,0 +1,113 @@
+<?php
+
+namespace Opifer\MailingListBundle\Block\Service;
+
+use Symfony\Component\DependencyInjection\ContainerInterface as Container;
+use Doctrine\ORM\EntityManagerInterface;
+use Symfony\Component\HttpKernel\Log\LoggerInterface;
+use Opifer\MailingListBundle\Entity\MailingList;
+use GuzzleHttp\Client as GuzzleClient;
+use GuzzleHttp\HandlerStack;
+use GuzzleHttp\Subscriber\Oauth\Oauth1;
+
+class ContactsManager
```

Comment:
Can you please call this something like SubscriptionManager? Since the entity is called Subscription.